packages/fether-react/src/Whitelist/NewTokenItem/NewTokenItem.js

```diff
@@ -4,31 +4,27 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 import React, { Component } from 'react';
-import { inject, observer } from 'mobx-react';
+import { consumeTokens } from '../../contexts/TokensContext.js';
```

Comment:
A small grumble: for consistency let's put relative imports below absolute ones with a line break (and ordered alphabetically). There are a couple of places like this in the PR.

Though I tend to put React first always, which kind of breaks the rule, but seems more comfortable.

Reply:
ordered alphabetically based on the package name or first named import?

for example in `App.js` there is
```
import Accounts from '../Accounts';
import Onboarding from '../Onboarding';
import Overlay from '../Overlay';
import { STATUS } from '../stores/healthStore';
import Send from '../Send';
import Tokens from '../Tokens';
import Whitelist from '../Whitelist';
```